BREAKING_CHANGES.md

```diff
@@ -69,6 +71,30 @@ export const config: Config = {
 };
 ```
 
+#### Backwards-compat Configuration Fields Deprecated
+
```

Comment:
Can we make spell this out in full? 
```suggestion
#### Backwards Compatible Configuration Fields Deprecated

```

Maybe something slightly different to draw attention to the fact these are browser-specific?
```suggestion
#### Legacy Browser Support Fields Deprecated

```
Thoughts?

Reply:
ooh yeah legacy browser gets at it well 👍